supervision/detection/utils.py

```diff
@@ -48,6 +48,36 @@ def polygon_to_mask(polygon: np.ndarray, resolution_wh: Tuple[int, int]) -> np.n
     return mask
 
 
+def box_iou(box1: np.ndarray, box2: np.ndarray) -> float:
```

Comment:
- [ ] Let's keep the signature of the function consistent with that of `box_iou_batch`.
- [ ] Let's also ensure that the boxes can be simply represented as lists and well as `np.ndarray`s.


```suggestion
def box_iou(boxes_true: Union[List, np.ndarray], boxes_detection: Union[List, np.ndarray]) -> float:
```